core/i18n/resources/en.js

```diff
@@ -145,7 +146,18 @@ export const enTranslation = {
     createSurvey: 'Create Survey',
     surveyDeleted: 'Survey {{surveyName}} has been deleted',
     surveyInfo: {
-      confirmPublish: `Do you want to publish this survey? Some operation won't be allowed afterwards.`,
+      confirmPublish: `**You are about to publish the survey {{survey}}**
+      
+Please, note that:
+
+- All labels associated with deleted languages will be permanently deleted.
+- Node definitions cannot be changed from single to multiple and vice versa.
+- All category items cannot change code and cannot be deleted.
+- All taxa cannot be modified.
+- All records associated with deleted cycles will be permanently deleted.
+- All nodes associated with deleted node definitions will be permanently deleted.
+
```

Comment:
Maybe rephrase as follows?

```
- Category items' codes cannot be changed after publishing
- Category items cannot be deleted after publishing
- Existing taxa cannot be modified after publishing
```

Although I'm not sure about the taxa. Can new taxa be added? Can they still be deleted?

Also, is "node" something that is obvious to the end user? I haven't seen "node" mentioned anywhere in the UI except for error messages.

Reply:
I agree with Elias, you should specify "after publishing" somewhere in the message, maybe in the beginning of it: Please not that after publishing: ...

Reply:
how about this

```#### You are about to publish the survey {{survey}} ####

###### During publish: ###### 
- All labels associated with deleted languages will be permanently deleted.
- All records associated with deleted cycles will be permanently deleted.
- Data associated with deleted form fields will be permanently deleted.

###### After publish: ###### 
- Form fields cannot be changed from single to multiple and vice versa.
- Category items' codes cannot be changed.
- Category items cannot be deleted.
- Taxa cannot be modified.

** Are you sure you want to continue? **```

Reply:
i changed the message